fsw/src/crypto.c

```diff
@@ -2647,46 +2648,252 @@ static int32 Crypto_PDU(char* ingest,TC_t* tc_frame)
 
     return status;
 }
-
-int32 Crypto_TC_ApplySecurity(char** ingest, int* len_ingest)
+int32 Crypto_TC_ApplySecurity(const uint8* in_frame, const uint32 in_frame_length, \
+    uint8 **enc_frame_test, uint32 *enc_frame_len)
```

Comment:
Let's refactor "_test" out of the function parameters

Reply:
Addressed in upcoming pull request. Minor changes to function variable names to improve code readability:

int32 Crypto_TC_ApplySecurity(const uint8* p_in_frame, const uint16 in_frame_length, \
    uint8 **pp_in_frame, uint16 *p_enc_frame_len)

Reply:
[cutright_dev f9535fe]